meta-refkit-core/recipes-ostree/refkit-ostree/refkit-ostree_0.0.bb

```diff
@@ -0,0 +1,42 @@
+SUMMARY = "IoT RefKit ostree helper, scripts, services, et al."
```

Comment:
I guess it is possible to switch back from a non-versioned to a versioned recipe in case it turns out later we want/need to start versioning, right. Then, we can safely change to a non-versioned recipe. I'll include that in an update PR.